scripts/data_processing/run_battlemap_pipeline.py

```
@@ -252,14 +328,56 @@ def show_config_summary(config: PipelineConfig):
+            max_tiles_per_image=config_data.get("max_tiles_per_image"),
+            boring_threshold=config_data.get("boring_threshold", 0.5),
+            output_dir=config_data.get("output_dir", output_dir),
+            tile_output_size=config_data.get("tile_output_size", 512),
+            temp_dir=config_data.get("temp_dir"),
+            save_progress=config_data.get("save_progress", True),
+            debug=config_data.get("debug", True),
+            test_mode=config_data.get("test_mode", False),
+            use_multiprocessing=config_data.get("use_multiprocessing", True),
+            clear_hash_cache=config_data.get("clear_hash_cache", True),
+        )
+        return config
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)